doc: worker_manager use suggestion to reduce boilerplate (#10224)
flutter/website: src/perf/isolates.md

or if you have a process that runs over a period of time
and needs to yield multiple return values to the main isolate.

> Or, you can try to use [worker_manager](https://pub.dev/packages/worker_manager) to simplify your long-lived isolates managing.

Reviewer: ```suggestion
Or, you might use [worker_manager][] to manage long-lived isolates.

[worker_manager]: {{site.pub-pkg}}/worker_manager
```
